samples/stock/demo/inverted-navigator/demo.js

```diff
@@ -0,0 +1,457 @@
+const eventsData = [{
+    x: -11494347287000,
+    y: 1,
+    lbl: 'The newspaper',
+    location: 'Strasbourg',
+    details: 'The first newspaper was invented by Johann Carolus. It was named <i>Relation aller Fürnemmen und gedenckwürdigen Historien</i> (Account of all distinguished and commemorable stories). The Relation is recognised by the World Association of Newspapers, as well as many authors, as the world\'s first newspaper.'
```

Comment:
Instead of `lbl`, I suggest using the term `name`, which is an actual part of the `PointOptionsObject` type. 

As for `location` and `details`, we could move them to a `custom` object. That way the data will be fully typesafe, making it easier for us and our clients when we move to TypeScript-based demos down the line.